orientation-event/motion/permissions-policy/devicemotion-allowed-by-permissions-policy-attribute-redirect-on-load.https.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<body>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/resources/testdriver.js"></script>
+<script src="/resources/testdriver-vendor.js"></script>
+<script src="/permissions-policy/resources/orientation-event.js"></script>
+<script src="/permissions-policy/resources/permissions-policy.js"></script>
+<script src="/common/get-host-info.sub.js"></script>
+<script src="../../resources/orientation-event-helpers.js"></script>
+<script>
+"use strict";
+
+const relative_path =
+  "/permissions-policy/resources/permissions-policy-orientation-event.sub.html?eventName=devicemotion";
+const base_src = "/permissions-policy/resources/redirect-on-load.html#";
+const same_origin_src = base_src + relative_path;
+const cross_origin_src =
+  base_src + get_host_info().HTTPS_REMOTE_ORIGIN + relative_path;
+
+promise_test(async t => {
+  const helper = new SensorTestHelper(t, 'devicemotion');
+  await helper.grantSensorsPermissions();
+  await helper.initializeSensors();
+
+  const motionData = generateMotionData(1, 2, 3,
+                                        4, 5, 6,
```

Comment:
Nit: can we run prettier over this? Otherwise we use up with inconsistent/arbitrary whitespace usage and closing brackets etc.

Reply:
Do you mean [this prettier](https://prettier.io/)? I'm fine with doing that, I just can't find any references to it in the tree or which settings should be used (I see a lot of tunables in https://prettier.io/playground/).

Reply:
I just use the defaults when I run it in Visual Studio Code.

Reply:
I've run prettier on all files here. PTAL.